ledger/cow.go

```diff
@@ -204,6 +214,10 @@ func (cb *roundCowState) put(addr basics.Address, new basics.AccountData, newCre
 	}
 }
 
+func (cb *roundCowState) trackCreatable(creatable basics.CreatableLocator) {
```

Comment:
Just pass in the index, now that you don't need the Locator

Reply:
+1 to JJ